release-notes.txt

```diff
@@ -1,3 +1,46 @@
+OpenMDAO Version 1.6.0 Alpha Release Notes
+March 1, 2016
+
+Backwards-Incompatible Change
+*System names must be valid Python names:
+ Because we add subsystems as attributes of their parent system, the name of each system must be a valid Python name. Otherwise we will raise an exception.
+
+Features
+*User now has the ability to call check_total_derivs and have it just do the driver-relevant params/cons/objs instead of every param/unknown in the model.
+*Multiple local components are now allowable in a parallel system.
+*User can complex step for total derivatives.
+*Created an NPSS wrapper for OpenMDAO 1.0.
+*We now account for Jacobian sparsity patern that arises from src_indices connections.
+*Documented more of our MPI functionality.
+*Added support for File variables.
+*Made the case-recording default to save derivatives.
+*Add unit conversion capability to pass-by-object.
+*Now raise an error if a user tries to add the same constraint/param twice.
+*Made Brent solver work for when state variable is an element of an array variable.
+*Sped up setup of model test nsga2 with pyoptsparse wrapper.
+*Added dynamic load balancing option for PredeterminedRunsDriver.
+*Removed getitem() function from System class.
+*Added complex step example to the FD tutorial.
+*Edited paraboloid tutorial and sellar tutorial to include lines where we set initial guesses.
+
+Bugfixes
+*Components without outputs now allow linearize to return an empty dict ({}).
+*User can now specify units for ExecComp values.
+*Fixed a bug in pyopt/SNOPT wherein it requires at least one nonlinear constraint.
+*Two_sided_constraints are now supported in ScipyOptimizer.
+*Unit conversions were being reported for pass-by-object variables.
+*Fixed bug in check_setup when connecting array_src to scalar target.
+*Cleared up problems that ExternalComp had with timeout and Windows process termination.
+*check_total_derivatives did not work under single_voi_relevance_reduction because the vectors were setup for only fwd or rev.
+*DOE des-vars with array variables didn't work.
+*optlatinhypercube was missing the load_balance arg.￼ 
+*Fixed bug where RBF interpolator returned NaN when a Jacobian is requested at a point in the training set.
+*Group Finite-Difference with pass-by-object.
```

Comment:
Fixed a bug where finite difference of a group would error out if there was a pass-by-object param or unknown in any sub-component.